fe1-web/.eslintrc.js

```diff
@@ -103,7 +103,5 @@ module.exports = {
     // enforce the use of color variables. this should make programmers use the defined color palette
     // or extend it if really necessary but not add new colors for every new component
     "react-native/no-color-literals": 'error',
-    // react native cannot handle text outside of <Text/> components. this only works on the web
-    "react-native/no-raw-text": 'error',
```

Comment:
Unfortunately this leads to false positives if some other component already wraps its children in a <Text/>.